llvm/test/Transforms/SimplifyCFG/switch-of-powers-of-two.ll

```diff
@@ -0,0 +1,90 @@
+; RUN: opt %s -passes='simplifycfg<switch-to-lookup>' -simplifycfg-require-and-preserve-domtree=1 -S | FileCheck %s
```

Comment:
You can use `llvm/utils/update_test_checks.py` to generate tests.

Reply:
Done. Thanks